.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: java
-sudo: required
+os: linux
 git:
   depth: false
 services:
 - docker
 jdk:
-- oraclejdk8
+- oraclejdk9
```

Comment:
why you are upgrading to a not-supported 9 version?

Reply:
Damn, I thought I've rolled this back. Sorry, will fix in about an hour